pages/linux/ip-link.md

```diff
@@ -0,0 +1,28 @@
+# ip link
+
+> Manage network interfaces.
+> More information: <https://man7.org/linux/man-pages/man8/ip-link.8.html>.
+
+- Show information about all network interfaces:
+
+`ip link`
+
+- Show information about a specific network interface:
+
+`ip link show {{ethN}}`
+
+- Bring a network interface up or down:
+
+`ip link set {{ethN}} {{up|down}}`
+
+- Give a meaningful name to a network interface:
+
+`ip link set {{ethN}} alias {{"LAN Interface"}}`
```

Comment:
The quotes should be outside:
```suggestion
`ip link set {{ethN}} alias "{{LAN Interface}}"`
```

Reply:
Do we have that in the guidelines?  Ought to.

Reply:
Maybe not.

Reply:
Nope, I'll do that today.

Reply:
The quotation marks aren't a mandatory part of the syntax. They're only required in this case because the example name contained a space. An equivalent syntax would be to escape the space with a backslash, which would appear within the braces.

Reply:
Oh.  Can you give an example similar to `ethN` then?

Reply:
Do you mean something like {{meaningful_name}}? I was trying to convey in the example that it's OK to use spaces in the name as long as the name is quoted.

Reply:
Ah I see, I wasn't being clear.  From the man page:

>give the device a symbolic name for easy reference

So it doesn't matter what name you give it. I think `LAN Interface` is fine then and the quotes are fine as well.  Just put them outside of the token syntax.  The unspoken rule is that if an argument _can_ have spaces in it, put quotes around it, even if the specific example of the argument that you're writing doesn't have spaces in.

Makes sense?

Reply:
OK. 

Thanks again for your patience walking me through the process as I'm learning.

Reply:
@aherst no problem whatsoever! 💯